js/compiler.ts

```diff
@@ -649,8 +649,8 @@ export class DenoCompiler
     return moduleNames.map(name => {
       let resolvedFileName;
       if (name === "deno" || name === "compiler") {
-        // builtin modules are part of `globals.d.ts`
-        resolvedFileName = this._resolveModuleName("globals.d.ts", ASSETS);
+        // builtin modules are part of `lib.d.ts`
+        resolvedFileName = this._resolveModuleName("lib.d.ts", ASSETS);
```

Comment:
1. 👍 
2. In regards to earlier comments, it would assume that our downstream unit tests provide sufficient coverage.  For example `tests/error_003_typescript.ts` ensure this _has_ to be working for there to be an error and other `.ts` tests _must_ be authored in a type consistent way for them not to throw.  Again, not the best way to test.
3. Covered by #876 now.